networkx/algorithms/__init__.py

```diff
@@ -1,3 +1,4 @@
+from networkx.algorithms.applications import *
```

Comment:
Yes sure. Othewise, problems of the same category should be placed in the same package.